app/controllers/device_stories_controller.rb

```diff
@@ -24,7 +24,7 @@ def show
   def get_like_searched_table(search_term)
     apply_scopes(DeviceStory).where('lower(device_urn) LIKE :search OR lower(custodian_name) LIKE :search', search: "%#{search_term}%")
       .or(apply_scopes(DeviceStory).where('lower(last_values) LIKE :search OR lower(last_location_name) LIKE :search', search: "%#{search_term}%"))
-      .or(apply_scopes(DeviceStory).where('CAST(last_seen AS text) LIKE ?', "%#{search_term}%")).page(params[:page]).per(params[:per_page])
+      .or(apply_scopes(DeviceStory).where('CAST(last_seen AS text) LIKE ?', "%#{search_term}%")).page(params[:page]).per(100)
```

Comment:
Not sure this is what @robouden was looking for. Seems like he was aiming for being able to see many (1000+) rows per page.

I'm fine with having that option (looks like we do already via the per_page query param), just make sure the default is more like 100. Otherwise people casually flipping through the site will cause unnecessary DB load.

Reply:
Does it make sense to keep what we had as default and to add a button that puts all results on one page? The button would set the `per_page` parameter to something like `DeviceStory.count`.